<?xml version="1.0"?>
<launch>
    <arg name="cameraTopics" />

    <node pkg="navigation_test_analysis" type="multiImageView.py" name="map_to_image" output="screen">
        <param name="cameraTopics" value="$( arg cameraTopics )" />
    </node>

</launch>
